snuba/clickhouse/translators/snuba/allowed.py

```diff
@@ -148,7 +149,7 @@ class DefaultNoneColumnMapper(ColumnMapper):
     the discover dataset file.
     """
 
-    columns: ColumnSet
+    columns: set[str]
```

Comment:
1. Can we make this a list and then check uniqueness in the `__post_init__` the yaml syntax for sets is pretty ugly
2. Can we fix the docstring and explain that the columns list is a list of strings mapping to the column names?